### PR TITLE
Feeds

### PR DESCRIPTION
**Pull request by sarah-arrrgh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,27 +0,0 @@
-# Logs
```
Comment (don-smith) [inline .gitignore:1]: I removed this file originally thinking it didn't make much sense in the `gh-pages` branch, but given how we're now using `master` and feature branches, I'm not sure that was a good idea. :(

Comment (sarah-arrrgh) [inline .gitignore:1]: Shall I add it in?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Project Positive</title>
+    <link rel="stylesheet" type="text/css" href="index.css">
+    <script type="text/javascript" src="./js/index.js"></script>
+    <script type="text/javascript" src="./js/instafeed.min.js"></script>
```
Comment (don-smith) [inline index.html:6]: These script references should be swapped based on potential dependencies?

Comment (sarah-arrrgh) [inline index.html:6]: Yes good point.

File: readme.md [hunk 3]
```diff
@@ -1,4 +1,25 @@
...
+
+* [Github](https://github.com/enspiral-craftworks/project-positive/blob/gh-pages/index.html)
+
+* [http://project-positive.org.nz/](http://project-positive.org.nz/)
+
+* [Facebook](www.facebook.com/heyawesome)
+
+* [Instagram](www.instagram.com/projectpositivenz)
+
+* [Twitter](www.twitter.com/projectposnz)
```
Comment (don-smith) [inline readme.md]: The links for FB, Instagram, and Twitter need `https://` in order to work.

Comment (sarah-arrrgh) [inline readme.md]: Done!

**Pull request closed without merging**